src/aasthaJJun20/SumOfAllDigitsInString.java

```diff
@@ -0,0 +1,23 @@
+package aasthaJJun20;
+/*Assignment 25 : 8th July 2020
+
+WAP to sum all the numbers in given string.
+Input: J7yu9y8h1h8j4b7j3jjb6
+Output: Sum of all numbers in String is: 53*/
+
+public class SumOfAllDigitsInString{
+	void findSumOfAllDigits(String input){
+		int sum = 0;
+		for(int index=0; index < input.length(); index++){
+			if(Character.isDigit(input.charAt(index))){
+				sum += Integer.parseInt(String.valueOf(input.charAt(index)));
```

Comment:
we can use Character class method here to convert char to int.

Character.getNumericValue(ch)